test/torchaudio_unittest/rnnt/rnnt_loss_impl.py

```diff
@@ -27,6 +27,12 @@ def test_basic_backward(self):
         loss = rnnt_loss(logits, targets, logit_lengths, target_lengths)
         loss.backward()
 
+    def test_basic_forward_no_grad(self):
```

Comment:
Can we give the description of the context?
I think for future maintainer, this test looks insignificant so it's better to give explanation why this test matters.
Something like

```
In early stage, `RNNTLoss` caused segmentation fault when `logits` have `requires_grad = False`.
This test makes sure that this segmentation fault is caught properly.

See also https://github.com/pytorch/audio/pull/1707
```

Reply:
I've added a comment in the following PR [here](https://github.com/pytorch/audio/pull/1711/commits/59366cc616f4adced46c3eed11367023e5cbc0d3#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR459)